materials/custom-properties.css

```diff
@@ -30,6 +30,29 @@
   --color-purple: #b866ff;
   --color-black: #1a1a1a;
   --color-white: #fff;
+  --color-primary: #00b39e;
+  --color-primary-25: hsl(172.9608938547486, 100%, 25%);
+  --color-primary-40: hsl(172.9608938547486, 100%, 40%);
+  --color-primary-85: hsl(172.9608938547486, 100%, 85%);
+  --color-primary-95: hsl(172.9608938547486, 100%, 95%);
```

Comment:
`172.9608938547486` 👀

Reply:
AFAIK these values haven't changed 😆. They are just copy pasted of `color-green-25`, etc. But yeah, the weird `172.9608938547486` value comes because before we were using some postcss plugin to generate the value. Might make sense to round these 🤔

Reply:
I guess rounding them would be a separate task.

Reply:
Or maybe use rgba instead of hsl?